TypeCobol/AntlrGrammar/Cobol2002CodeElements.g4

```diff
@@ -44,6 +45,13 @@ initializeStatement:
 	(THEN? REPLACING initializeReplacingDirective+)?
 	(THEN? TO? DEFAULT)?;
 
+// New Cobol v6.1 ALLOCATE statement to obtain dynamic storage.
+// 'INITIALIZED' is defined here as a contextual keyword. The storageArea2 is therefore not allowed to be named 'INITIALIZED' in this statement.
+allocateStatement:
+	ALLOCATE ((arithmeticExpression CHARACTERS) | { !string.Equals(CurrentToken.Text, "INITIALIZED", System.StringComparison.InvariantCultureIgnoreCase) }? storageArea2)
```

Comment:
It doesn't works like that ? 
```
	ALLOCATE ((arithmeticExpression CHARACTERS) | storageArea2)
	({ string.Equals(CurrentToken.Text, "INITIALIZED", System.StringComparison.InvariantCultureIgnoreCase) }? KeywordINITIALIZED=UserDefinedWord)?
	(RETURNING pointerStorageArea)?;
```

Reply:
It does but it will successfully parse 'ALLOCATE INITIALIZED' statement which is incorrect. storageArea2 is not allowed to be named 'INITIALIZED' in this context as we cannot distinguish between the data name and the optional keyword.
The COBOL compiler rejects 'INITIALIZED' as an identifier so it seems more robust to forbid it in this case.